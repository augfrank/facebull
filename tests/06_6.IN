M0   C0   C1  2952
M1   C0   C3  2109
M2   C0   C4  2732
M3   C0   C5  2344
M4   C1   C0  2313
M5   C1   C2  2321
M6   C1   C3  2302
M7   C2   C0  2856
M8   C2   C1  2869
M9   C2   C3  2187
M10   C3   C1  2086
M11   C3   C2  2597
M12   C3   C4  2685
M13   C4   C0  2207
M14   C4   C1  2557
M15   C4   C2  2919
M16   C4   C3  2280
M17   C4   C5  2258
M18   C5   C0  2837
M19   C5   C1  2481
M20   C5   C2  2996
M21   C5   C3  2747
