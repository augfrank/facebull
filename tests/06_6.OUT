14225
3 5 9 12 13 19
